pkg/reconciler/knativeeventing/source/source.go

```diff
@@ -54,7 +54,7 @@ func getSourcePath(version string, ke *v1alpha1.KnativeEventing) string {
 
 	// This line can make sure a valid available source version is returned.
 	sourcePath := filepath.Join(koDataDir, "eventing-source", sourceVersion)
-	urls := make([]string, 0, 100)
+	urls := make([]string, 0)
```

Comment:
I always wonder if `urls := []string{}` are better style. But either way.

Reply:
In this case, it should actually be `var urls []string` to avoid an allocation if there's no source enabled.

/hold

Reply:
Not that it matters literally at all, but I’d do `var urls []string` here, the difference is incredibly small, but the var version doesn’t needlessly create the array if it doesn’t end up getting used.

Reply:
heh oops, @markusthoemmes said the same thing :)